src/apis/drive/samples/node/quickstart.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018, Google, LLC.
```

Comment:
It's unclear to me if this file would be generated or manual?

Reply:
Manual - just like now. Samples will exist only for those APIs they exist for now.